src/sections/footer.html

```diff
@@ -1,4 +1,26 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<div class="px-7 relative mt-36">
+    <img src="./src/images/Vector_7.svg" class="mx-auto sticky top-0 z-30" alt="">
+
+    <div class="flex gap-7 sm:flex-col lg:flex-row flex-col lg:justify-between md:gap-10 md:text-2xl font-medium md:mt-40 mb-4 mt-20">
+        <div class="flex flex-col lg:gap-28 justify-between md:flex-row gap-7 ">
+            <div class="flex flex-col ">
```

Comment:
`<div class="flex flex-col ">`
replace with:
`<div class="flex flex-col">`

The reason is to be constant in the spaces you put in your codes.